src/analysis/rule_extraction.jl

```diff
@@ -129,8 +132,24 @@ A StableRules object (implemented by SIRUS).
    Electron. J. Statist. 15 (1) 427 - 505.
    https://doi.org//10.1214/20-EJS1792
 """
-function cluster_rules(clusters::Vector{T}, X::DataFrame, max_rules::T;
-    seed::Int64=123, kwargs...) where {T<:Int64}
+function cluster_rules(
+    result_set::ADRIA.ResultSet,
```

Comment:
Minor comment - could this be simplified to `ResultSet`? If not, maybe we should consider exporting the `ResultSet` type...

Reply:
Yep, that works.